### PR TITLE
[plugin.video.tweakers] 1.1.9

### PR DESCRIPTION
**Pull request by skipmodea1**
### Description
v1.1.9 (2017-12-28)
- removed looking for video dialogue
- fix to match the correct thumbnail with the item
- addon now works in kode python 2 and also 3 (!!) thanks to the future project
- addon.xml contains NOT YET SUPPORTED python3 tag
### Checklist:

- [ ] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [ ] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [ ] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.tweakers/addon.xml [hunk 1]
```diff
@@ -2,12 +2,14 @@
...
-	version="1.1.8"
-	provider-name="Skipmode A1,Amelandbor,ninovanhooff,Martijn">
+	version="1.1.9"
+	provider-name="Skipmode A1">
   <requires>
-    <import addon="xbmc.python"                 version="2.14.0"/>
-	<import addon="script.module.beautifulsoup" version="3.0.8"/>
-    <import addon="script.module.requests"      version="2.4.3"/>
+    <import addon="xbmc.python"                     version="2.14.0"/>
+	<import addon="script.module.beautifulsoup4"    version="4.5.3"/>
```
Comment (Rechi) [inline plugin.video.tweakers/addon.xml]: script.module.beautifulsoup4 version for gotham is 4.3.2

File: plugin.video.tweakers/addon.xml [hunk 2]
```diff
@@ -27,5 +29,6 @@
     <website>http://www.tweakers.net</website>
     <email></email>
     <source>https://github.com/skipmodea1/plugin.video.tweakers</source>
+    <python3>true</python3>
```
Comment (Razzeee) [inline plugin.video.tweakers/addon.xml]: Have we decided on this format yet? Feels a bit premature to include?

Comment (MartijnKaijser) [reply plugin.video.tweakers/addon.xml]: Nothing decided yet

Comment (skipmodea1) [reply plugin.video.tweakers/addon.xml]: Right, i'll close the PR and wait for a decision.

Comment (Rechi) [reply plugin.video.tweakers/addon.xml]: why not update the addon without this?

**Pull request closed without merging**